test/regressions/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@mui-x-internal/test-regressions",
+  "version": "0.0.0",
+  "private": true,
+  "type": "module",
+  "scripts": {
+    "start": "cross-env NODE_ENV=production pnpm build && concurrently --success first --kill-others \"pnpm run mocha\" \"pnpm run server\"",
+    "build": "cross-env E2E_BUILD=true vite build",
```

Comment:
If `E2E_BUILD` is supposed to be used in both e2e and regression tests, it's better to rename it to `TEST_BUILD` to avoid confusion.